dev/devicelab/lib/tasks/plugin_tests.dart

```diff
@@ -315,7 +315,7 @@ public class $pluginClass: NSObject, FlutterPlugin {
   Future<void> runNativeTests(String buildTarget) async {
     // Native unit tests rely on building the app first to generate necessary
     // build files.
-    await build(buildTarget, validateNativeBuildProject: false);
+    await build(buildTarget, validateNativeBuildProject: false, buildMode: 'debug');
```

Comment:
Previously there was a mixture of release and debug here. That broke Xcode tests. I made everything debug.